tests/logictest/suites/query/alias/case.test

```diff
@@ -0,0 +1,28 @@
+statement ok
+set enable_planner_v2 = 1;
+
+statement ok
+create database db;
+
+statement ok
+create table db.t(a int);
+
+statement ok
+create table db.t1(a int);
+
+statement ok
+insert into db.t values(1);
+
+statement ok
+insert into db.t1 values(2);
+
+onlyif mysql
+statement query II
+select db.t.a as a, t1.a from db.t left join db.t1 "T1" on db.t.a <> t1.a;
```

Comment:
Cloud we use `T1` to replace "T1"? Because some Bi will auto-generate SQL like `T1`.

Reply:
![image](https://user-images.githubusercontent.com/22445410/177271068-5e5c8b1d-895d-47f8-81eb-59b0d15f0f9f.png)

We can do that.